README.md

```diff
@@ -99,5 +99,6 @@ See the documentation contained in the docs folder:
 3. [Usage guide](docs/usage-guide.md)
 4. [Developer guide](docs/developer-guide.md)
 5. [Design](docs/design.md)
-6. [Common problems and their solutions](docs/common-problems-and-their-solutions.md)
-7. [Roadmap](docs/development/roadmap.md)
+6. [Design risks and mitigations](docs/development/design-risks-and-mitigations.md)
```

Comment:
If we link this directly from the readme it might be worth keeping it in the root of the docs folder, to be consistent with the other documents.

Reply:
Done